CHANGELOG.md

```diff
@@ -0,0 +1,87 @@
+# Changelog
+
+All notable changes to this project will be documented in this file (starting with 1.0.0).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - yyyy-mm-dd (in UTC)
+
+### Added
+
+### Changed
+
+### Removed
+
+
+## [Unreleased - 1.0.0 ] - 2025-07-xx
+
+The first "stable" release following semver but not extensively other than the previous releases. The use of 1.0 indicates the stability that users already expect from InvenTree.
+
+An overarching theme of this release is the complete switch to a new UI framework and paradigm (PUI). The old templating based UI (CUI) is now removed. This makes major improvements in the security and portability of InvenTree possible.
+
+Our blog holds [a few articles](https://inventree.org/blog/2024/09/23/ui-roadmap) on the topic. This journey started in [March 2022](https://github.com/inventree/InvenTree/issues/2789) and was announced [in 2023](https://inventree.org/blog/2023/08/28/react).
+
+### Added
+
+
```

Comment:
If we are going to make this a thing from 1.0.0 onwards, should we remove this section here (I don't see it being filled out...)

Reply:
You mean added features would only be seen in the git-commit based changelog?

Reply:
No, I just mean in the file you have submitted, removing the sections *before* the 1.0.0 release which will remain empty anyway?